sample.json

```diff
@@ -0,0 +1,15 @@
+{
+  "uuid" : "path/to/example.tif",
```

Comment:
@lossyrob I don't think we're thinking of separating out the image and data files. They should colocate. Can't the UUID and URI for the image be the same?